From 6d032421b4871105e1f37bb75a5ec7b012f19f61 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sun, 15 Jul 2012 22:30:07 -0400
Subject: [PATCH] gio: use /usr/share/bash-completion for bashcomp

In Gentoo, we use /usr/share/bash-completion, not
/usr/share/bash-completion/completions.
---
 gio/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gio/Makefile.am b/gio/Makefile.am
index 704367c..bee9dc6 100644
--- a/gio/Makefile.am
+++ b/gio/Makefile.am
@@ -715,7 +715,7 @@ gdbus_LDADD = libgio-2.0.la \
 	$(top_builddir)/glib/libglib-2.0.la 		\
 	$(top_builddir)/gobject/libgobject-2.0.la
 
-completiondir = $(datadir)/bash-completion/completions
+completiondir = $(datadir)/bash-completion
 completion_DATA = \
 	completion/gdbus				\
 	completion/gsettings				\
-- 
1.7.8.6

